src/figuresClasses.ts

```diff
@@ -1,19 +1,83 @@
+type Shapes = 'triangle' | 'circle' | 'rectangle';
+type Colors = 'red' | 'green' | 'blue';
+
 export interface Figure {
+  shape: Shapes;
+  color: Colors;
+  getArea(): number;
+}
 
+function getRoundNumber(area: number): number {
+  return Math.floor(area * 100) / 100;
 }
```

Comment:
you don't just round the number here, let's be more specific
```suggestion
function getRoundedDownArea(area: number): number {
  return Math.floor(area * 100) / 100;
}
```